pkgs/development/tools/convenient_test_manager/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, flutter
+, makeDesktopItem
+, libvlc
+, copyDesktopItems
+}:
+
+flutter.mkFlutterApp rec {
+  pname = "convenient-test-manager";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "fzyzcjy";
+    repo = "flutter_convenient_test";
+    rev = "v${version}";
+    sha256 = "sha256-f0+nATpBAd+3ELRq7dfKi9HCawgF3LAAs0UqYjnKCPw=";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = "${placeholder "out"}/bin/convenient_test_manager";
+    icon = "convenient_test_manager";
+    desktopName = "Convenient Test Manager";
+    genericName = "Flutter test runner";
+    categories = [ "Development" ];
+  };
+
+  vendorHash = "sha256-iR/U2vQQqj95j1XkgW3/u7DQzwQrNmho9M4yLvefb4A=";
```

```suggestion
  vendorHash = "sha256-GImFKbhAUOZn2R7274HqHaS3gq4bt/rhCUL+B+q2uXY=";
```

that hash is unstable. Can you check why that is the case?

---

drafting it because of this